### PR TITLE
W-11021541 Adding Alt. Text

### PR DESCRIPTION
**Pull request by hlouis-mulesoft**

### CODE REVIEW

File: modules/ROOT/pages/create-waf-policy.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ include::partial$anypoint-security-policies-prereqs.adoc[]
 . Enter a *Name* for the policy and click *Save Policy*. +
 The policy appears in the *Security Policies* list, where you can edit and delete policies.
 
-image::security-policy-list.png[]
+image::security-policy-list.png[Blue Create Policy button located above the table of security policies which lists the name, type, and action.]
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-waf-policy.adoc]: When there are commas in the alt text, use "".
```suggestion
image::security-policy-list.png["Blue Create Policy button located above the table of security policies, which lists the name, type, and action."]
```

File: modules/ROOT/pages/create-waf-policy.adoc [hunk 2]
```diff
@@ -32,7 +32,8 @@ Information about the violation is also sent to the log, per incident, at `INFO`
 +
 When you hover over the `i` to the right of each rule, the rule ID range for that ruleset is displayed. You can use this information for testing rulesets.
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-waf-policy.adoc:33]: When you hover over the information (i) icon for each rule, the rule ID range for that ruleset is displayed.

File: modules/ROOT/pages/create-waf-policy.adoc [hunk 3]
```diff
@@ -32,7 +32,8 @@ Information about the violation is also sent to the log, per incident, at `INFO`
...
 +
-image::waf-rule-set.png[]
+image::waf-rule-set.png[Rule ID range is displayed for Scanner detection as a pop-up message.]
+// not sure if this one is necessary; it adds something for a seeing user I guess but the alt text wouldn't really add anything new
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-waf-policy.adoc:36]: You're probably right, but let's leave it. Don't forget to delete your //comment before you merge.  :-)

File: modules/ROOT/pages/grant-tokenization-permissions.adoc [hunk 4]
```diff
@@ -24,5 +24,6 @@ A user must have a set of predefined permissions to create and manage tokenizati
 . In the Access Management page, go to the left navigation menu and click *Users*.
 . To grant permissions to a user, click the user's name. 
 . In the *Permissions* tab, click *Tokenization*. 
-. Select *Manage Tokenization Formats* from the *Permissions* menu, and click the blue "plus" icon image:blue-plus-security.png[]. 
-. Select the environment from the Environment menu, then select the *Manage Tokenization Services* permission from the Permissions menu, and click the blue "plus" icon image:blue-plus-security.png[].
\ No newline at end of file
+. Select *Manage Tokenization Formats* from the *Permissions* menu, and click the blue "plus" icon image:blue-plus-security.png[Small white plus insaide a larger blue circle].
```
Comment (valkyrie69) [inline modules/ROOT/pages/grant-tokenization-permissions.adoc]: You're right. Let's take this one out.

File: modules/ROOT/pages/index-policies.adoc [hunk 5]
```diff
@@ -7,7 +7,8 @@ Anypoint Security features a high-performance, reliable, and scalable service th
 
 You can use the Anypoint Security policies to manage all traffic to your Runtime Fabric, and leverage API Manager policies to apply specific behaviors to specific APIs.
 
-image::index-e9021.png[]
+image::index-e9021.png[Flow Chart connecting Anypoint Platform services with various Runtime Fabric API Policies and then to respective Mule applications.]
```
Comment (valkyrie69) [inline modules/ROOT/pages/index-policies.adoc]: It's okay to go a little over 125 characters, but if you want to be strict, you could do something like: 
```suggestion
image::index-e9021.png[Flow Chart connecting Anypoint Platform services with Runtime Fabric API Policies and their respective Mule applications.]
```

File: modules/ROOT/pages/index-secrets-manager.adoc [hunk 6]
```diff
@@ -38,7 +38,7 @@ This is an internal service that handles the secrets for consumption by supporte
 This is the only service that can read the actual secrets an due to its nature, it is not accessible from the public network. 
 
 
-image::secrets-manager-diagram.png[]
+image::secrets-manager-diagram.png[Diagram connecting Secrets Manager tp secret group to service provider to the supported Mule platform in Anypoint platform.]
```
Comment (valkyrie69) [inline modules/ROOT/pages/index-secrets-manager.adoc]: Let's try to simplify even further into something like:

```suggestion
image::secrets-manager-diagram.png[Diagram showing how secrets manager interacts with the secrets provider service and Anypoint Platform services.]
```

File: modules/ROOT/pages/tokenization.adoc [hunk 7]
```diff
@@ -17,7 +17,7 @@ Anypoint tokenization creates format-preserving tokens, which means the output t
 
 Making a detokenization call to the tokenization service returns the identity of a tokenized value.
 
-image::tokenization-flow.png[]
+image::tokenization-flow.png[Flow chart beginning with client information connecting to the API Gateway which has two paths: directly to the Mule App or a two-way connection with the tokenization service.]
```
Comment (valkyrie69) [inline modules/ROOT/pages/tokenization.adoc]:
```suggestion
image::tokenization-flow.png[Diagram showing how the client payload interacts with the tokenization policy and service to return a tokenized value to the Mule app.]
```

File: modules/ROOT/pages/waf-rulesets.adoc [hunk 8]
```diff
@@ -63,7 +63,7 @@ The Runtime Fabric policy summary includes the WAF summary. A WAF summary log me
 
 For example:
 
-image::waf-summary-log-raw.png[]
+image::waf-summary-log-raw.png[Difficult to read raw data from WAF summary log message.]
```
Comment (valkyrie69) [inline modules/ROOT/pages/waf-rulesets.adoc]:
```suggestion
image::waf-summary-log-raw.png[Unformatted raw data from WAF summary log message.]
```

File: modules/ROOT/pages/create-waf-policy.adoc (hunk 1 above)
Comment (hlouis-mulesoft) [reply modules/ROOT/pages/create-waf-policy.adoc]: Do I need to add quotations for all of the alt text?

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**